### PR TITLE
fix(docker): add packages to compile projects

### PR DESCRIPTION
**Pull request by PatrickMi**
Adds python, gcc, make and git to the docker container so it can compile its own needed binaries. 

closes https://github.com/maevsi/maevsi/issues/1279

### CODE REVIEW

**Comment by dargmuesli** [comment]
> Should I use past or present tense in git commit messages?

fyi: https://stackoverflow.com/a/3580764/4682621

**Comment by PatrickMi** [comment]
> Please add a separate Dockerfile for the arm architecture. Then, [digest-pinning](https://docs.renovatebot.com/docker/#digest-pinning) can be enabled again for each Dockerfile.

A second Dockerfile with the name Dockerfile.arm64-v8 was created. As shown in the Docker docs (https://docs.docker.com/engine/reference/commandline/build/#file) the prompt `sudo docker build -t maevsi/maevsi:dev --target development -f Dockerfile.arm64-v8 . ` can be used to build the ARM Dockerfile

File: src/Dockerfile.arm64-v8 [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+RUN pip3 install --no-cache --upgrade pip setuptools
+
+# Install make
+RUN apk add --no-cache make
+
+# Install gcc
+RUN apk add build-base
+
+# Install git
+RUN apk add git
```
Comment (dargmuesli) [inline src/Dockerfile.arm64-v8:22]:
```suggestion
# Workaround: install build tools for arm.
ENV PYTHONUNBUFFERED=1
RUN apk add --update --no-cache \
        python3 \
        make \
        build-base \
        git && \
    ln -sf python3 /usr/bin/python && \
    python3 -m ensurepip && \
    pip3 install --no-cache --upgrade pip setuptools
```

see https://github.com/hadolint/hadolint/wiki/DL3059

**Review by dargmuesli — CHANGES_REQUESTED** [review]
Also, I've pinned digests.

File: src/Dockerfile.arm64-v8 [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    pip3 install --no-cache --upgrade pip setuptools
+
+# Install make
+RUN apk add --no-cache make
+
+# Install gcc
+RUN apk add build-base
+
+# Install git
+RUN apk add git
```
Comment (dargmuesli) [inline src/Dockerfile.arm64-v8:26]:
```suggestion
```
Last suggestion was seemingly not applied as given.

**Review by dargmuesli — CHANGES_REQUESTED** [review]

**Review by dargmuesli — APPROVED** [review]
Please check if everything still works and merge if so!

**Pull request merged**